wsmaster/che-core-api-devfile/src/main/resources/schema/devfile.json

```diff
@@ -221,6 +221,48 @@
             "description": "Describes whether projects sources should be mount to the tool. `CHE_PROJECTS_ROOT` environment variable should contains a path where projects sources are mount",
             "default": "false"
           },
+          "command": {
+            "oneOf": [
+              {
+                "type": "array",
+                "items": {
+                  "type": "string"
+                }
+              },
+              {
+                "type": "string"
+              }
+            ],
+            "description": "The command to run in the dockerimage tool instead of the default one provided in the image.",
+            "examples": [
+              [
```

Comment:
Please check how it will affect Devfile Docs. See how Max defined map of strings https://github.com/eclipse/che/pull/12552/files#diff-abe34df16881c3a5f48a90721bbd7395R156

Reply:
Thanks, it should be fixed now.